spec.yaml

```diff
@@ -997,6 +997,58 @@ paths:
           schema:
             $ref: "./definitions.yaml#/definitions/V4GenericResponse"
 
+  /v4/organizations/{organization_id}/credentials/:
+    post:
+      operationId: addCredentials
+      tags:
+        - organizations
+      summary: Set credentials
+      description: |
+        Add a set of credentials to the organization allowing to create and
```

Comment:
"allowing to" sounds a bit strange to me. "allowing the creation and operation of clusters" is better IMO.